### PR TITLE
Issue 1 create an entry | Nick Hernandez

### PR DESCRIPTION
**Pull request by Nickolas-Hernandez**
Screen Shot 2021-02-08 at 5 18 15 PM
Screen Shot 2021-02-08 at 5 18 29 PM
Screen Shot 2021-02-08 at 5 19 17 PM

### CODE REVIEW

File: css/layout.css [hunk 1]
```diff
@@ -1,7 +1,3 @@
-.hidden {
-  display: none !important;
-}
-
```
Comment (codingcodymiller) [inline css/layout.css]: But why?

File: js/main.js [hunk 2]
```diff
@@ -1,2 +1,26 @@
...
+function handleEntrySubmit(event) {
+  event.preventDefault();
+  var entry = {
+    image: $entryForm.elements.image.value,
+    title: $entryForm.elements.title.value,
+    notes: $entryForm.elements.notes.value,
+    entryId: data.nextEntryId
+  };
+  data.nextEntryId++;
+  data.entries.push(entry);
```
Comment (codingcodymiller) [inline js/main.js]: This should be being prepended to the array, not appended to the end. You will need a different method than `push` here 🤓

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Great work so far, Nick! A couple notes for you:

File: css/layout.css (hunk 1 above)
Comment (Nickolas-Hernandez) [reply css/layout.css]: Whoops! not sure how that happened. Will fix

**Comment by Nickolas-Hernandez** [comment]
Made changes. See commit message. 
Screen Shot 2021-02-09 at 10 29 53 AM
Screen Shot 2021-02-09 at 10 30 19 AM
Screen Shot 2021-02-09 at 10 31 50 AM

**Review by codingcodymiller — APPROVED** [review]
🥳 🥳 🥳

**Pull request merged**